amy/consents/admin.py

```diff
@@ -34,18 +35,24 @@ def send_consent_email(request, term: Term) -> None:
         active=True,
         action="consent-required",
     )
+
+    emails_to_send = []
     for email in emails:
-        jobs, rqjobs = ActionManageMixin.add(
-            action_class=NewConsentRequiredAction,
-            logger=logger,
-            scheduler=scheduler,
-            triggers=triggers,
-            context_objects={
-                "term": term,
-                "person_email": email,
-            },
-            object_=term,
-        )
+        if len(emails_to_send) < settings.BULK_EMAIL_LIMIT:
+            emails_to_send.append(email)
+        else:
+            jobs, rqjobs = ActionManageMixin.add(
+                action_class=NewConsentRequiredAction,
+                logger=logger,
+                scheduler=scheduler,
+                triggers=triggers,
+                context_objects={
+                    "term": term,
+                    "person_email": emails_to_send,
+                },
+                object_=term,
+            )
+            emails_to_send = []
```

Comment:
I think we can calculate `emails_to_send` upfront:

```python
N = 1000
emails_to_send = [emails[i:i+N] for i in range(0, len(emails), N]
for email in emails_to_send:
    jobs, rqjobs = ActionManageMixin.add(...)
```

Reply:
Another thing: are there tests for this function? Should they be updated?

Reply:
@pbanaszkiewicz no I don't think there are. Good call out. I'll add some this weekend.